multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,25 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl
+    permissions: 0755
+    content: |
+      # source this file to set up a kubectl alias
+      sudo rkt run \
+        --net=host \
+        --volume bk,kind=host,source=/srv/kubernetes \
+        --mount volume=bk,target=/srv/kubernetes \
+        --insecure-options=image docker://{{.HyperkubeImageRepo}}:{{.K8sVer}} \
+        --exec /hyperkube -- kubectl $@
```

Comment:
@cgag 
> it does stuff like prevent glob expansion:

Yes, this is one example. Argument quoting controls the level at which **any subexpresion**, not just glob expansions, are expanded. Though that is entirely orthogonal to the argument parsing issue.

> I don't think that's how the quoting works.

Unfortunately that **is** how quoting works w.r.t argument parsing in bash, and your echo example does not address this behavior. Try this:

```sh
[chom@hompad-work ~]$ cat test.sh
#!/bin/bash

echo "arg count is: $#"

for arg in "$@";do
  echo "arg: \"${arg}\""
done

[chom@hompad-work ~]$ ./test.sh "larry moe curly"
arg count is: 1
arg: "larry moe curly"
[chom@hompad-work ~]$ ./test.sh larry moe curly
arg count is: 3
arg: "larry"
arg: "moe"
arg: "curly"
```

See what I mean?